src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/TableSaw.java

```diff
@@ -46,6 +46,11 @@ public TableSaw(Category category, SlimefunItemStack item) {
                 displayedRecipes.add(new ItemStack(planks.get(), 8));
             }
         }
+
+        for (Material plank: Tag.PLANKS.getValues()) {
+            displayedRecipes.add(new ItemStack(plank));
+            displayedRecipes.add(new ItemStack(Material.STICK, 4));
+        }
```

Comment:
Instead of flling the guide wth the same recipe maybe add a dummy plank representing all planks and have its display name as "Any Wooden Plank"

Reply:
Should I also do that for logs / stripped logs then, @LinoxGH?